blogposts/announcing-sourcegraph-3.1.md

```diff
@@ -0,0 +1,121 @@
+---
+title: 'Announcing Sourcegraph 3.1'
+author: 'Ryan Blunden'
+publishDate: 2019-02-20T05:59-06:00
+tags: [
+  "blog"
+]
+slug: sourcegraph-3.1
+heroImage: //images.ctfassets.net/le3mxztn6yoo/t4Qpcq5kA0AYM24Ws4mOk/4edf5502a936bbec90c262fa00355aed/sourcegraph-mark.png
+published: false
```

Comment:
Does this need to be true before merge?
```suggestion
published: true
```

Reply:
Yeah, was planning on changing it once it was ready.

Reply:
Isn't "ready" defined by "merged"?